src/app/components/wrapper-components/online-game-wrapper/online-game-wrapper.quarto.component.spec.ts

```diff
@@ -406,10 +488,16 @@ describe('OnlineGameWrapperComponent of Quarto:', () => {
         }, 1);
 
         // Then currentPart should not be updated
-        expect(wrapper.currentPart).toEqual(CURRENT_PART);
+        const lastUpdateTime: FirestoreTime | undefined = wrapper.currentPart.data.lastUpdateTime;
+        expect(lastUpdateTime === null ||lastUpdateTime === undefined).toBeTrue();
```

Comment:
Quand je met "== null" au lieu du === avec null&undefined ça me dit "Prefer jasmine matcher instead of comparisoneslint(jasmine/prefer-jasmine-matcher)"

Et j'fais pas confiance au toBeNull/toBeUndefined, celui là est explicit :p